kolena/dataset/evaluation.py

```diff
@@ -51,7 +51,14 @@
 from kolena.errors import NotFoundError
 
 EvalConfig = Optional[Dict[str, Any]]
+"""
+User defined configuration for evaluating results, for example `{"threshold": 7}`.
+"""
 DataFrame = Union[pd.DataFrame, Iterator[pd.DataFrame]]
+"""
+A type alias representing a DataFrame, which can be either a pandas DataFrame
+or an iterator of pandas DataFrames, data will be processed in chunks when iterator is used.
```

Comment:
Is `data will be processed...` needed? Seems `iterator` by itself is sufficient. (side nit, `in chunks` of iterator is confusing)

Reply:
how about we change it to `A type alias representing a DataFrame, which can be either a pandas DataFrame
or an iterator of pandas DataFrames`?

Reply:
Sounds good.